### ISSUE TITLE
M> First Meetup in Zagreb, Croatia about RChain

### ISSUE THREAD

**Issue by lapin7** [rchain/Members#125]
nathan is coordinating in New York.
ask about central meetup

**Comment by kitblake** [comment]
Also @jbassiri has created a useful generic RChain presentation, see #102.

**Comment by tgrospic** [comment]
Thanks @kitblake, I will definitely need it and use it.

**Comment by dckc** [comment]
I can't quite tell what the scope of this issue is. Nobody has touched it for months. If there is continued work to do, please clarify the scope / objective.

**Comment by tgrospic** [comment]
It's still work in progress. This will be the first introduction meetup in Zagreb, probably at the beginning of February. Should I open a new issue or reopen this one?

**Comment by dckc** [comment]
Thanks for clarifying. I couldn't even tell from the description that Zagreb was a place. I don't understand the connection to New York, but I suppose the relevant people do.

**Comment by Keaycee** [comment]
How about creating an event (pub Conference) about RChain in any part of the world and having souvenir, handout, leaflet and other form of advert being distributed around the place of event so we could enlighten more people and create aware across.

**Comment by lapin7** [comment]
@Keaycee good idea. Please make a SMART issue on event: where, when, how, etc.
What do you think of **RHOdes**, Greece ([GPS: 36,27](https://www.google.fr/maps/place/Rhodes/@36.1678912,27.6853208,10z/data=!3m1!4b1!4m5!3m4!1s0x14950af92e9b4fad:0x27eed014f85bd454!8m2!3d36.4340533!4d28.2176379?hl=en))
:-):-):-):-):-)

**Comment by dckc** [comment]
@Keaycee see https://github.com/rchain/Members/wiki/local-meetup-program:-summary for stuff common to many meetups.

**Comment by Keaycee** [comment]
I will look into that @dckc @lapin7

**Comment by pmoorman** [comment]
@tgrospic I'm going through all marketing issues, seeing if some can be closed.

Is this issue still relevant? 
Has the meetup successfully happened?

**If it's no longer relevant, then please close the issue.**

**Comment by InternautNeven** [comment]
i believe it is still on, i`ll contact @tgrospic TY

**Comment by tgrospic** [comment]
Yes, it's still relevant. @InternautNeven and @MarioPolo recently joined the team. They started translation of RChain whitepaper #449 and already done translation of Contributing document #443.
We are closer to our plan to organize monthly meetups. I also hope for useful information from the governance conference. :+1: 

We will update this issue with information about our first meetup so everyone can follow.

**Comment by pmoorman** [comment]
Ok, great!!

**Comment by InternautNeven** [comment]
Meetup will probably be in the begining of May, 03.05.2018. Thursday  here [ImpactHub] (https://zagreb.impacthub.net/ )  
We already made a [poster] (https://postimg.org/image/pxxridyrv/ ) (local artist)  and translating several documents #544 #382 #449
